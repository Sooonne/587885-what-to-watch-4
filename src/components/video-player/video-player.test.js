import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const video = {
  poster: `img/the-grand-budapest-hotel-poster.jpg`,
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
};

describe(`VideoPlayer`, () => {
  it(`VideoPlayer should render correctly`, () => {
    const tree = renderer
      .create(<VideoPlayer
        isPlaying={false}
        src={video.src}
        poster={video.poster}
      />, {
        createNodeMock: () => {
          return {};
        }
      })
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
