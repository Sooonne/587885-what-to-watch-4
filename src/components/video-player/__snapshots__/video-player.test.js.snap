// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer Should render correctly 1`] = `
<video
  height="175"
  muted={true}
  width="280"
/>
`;

exports[`VideoPlayer VideoPlayer should render correctly 1`] = `
<video
  height="175"
  muted={true}
  width="280"
/>
`;
